.changeset/sharp-mugs-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Added two new components, `Avatar` and `ImageInput`, to display and upload avatar images.
```

Comment:
Since the ImageInput is agnostic of the component that's used to display the image, I'd split this into two changesets.